libsolidity/interface/Natspec.h

```diff
@@ -40,6 +40,8 @@ struct DocTag;
 class Natspec
 {
 public:
+	static size_t constexpr NATSPEC_VERSION = 1;
```

Comment:
Don't we use this style only for preprocessor macros?

Reply:
Are you referring to the `constexpr` part or the `NATSPEC_VERSION` all-uppercase part? The former was from @leonardoalt suggestion, the later from our [style guide regarding constants](https://solidity.readthedocs.io/en/latest/style-guide.html#constants)

Reply:
https://github.com/ethereum/solidity/blob/develop/CODING_STYLE.md#4-variable-prefixes

I think it says prefix should be `s_`. Maybe camel case after that?

Reply:
It says `_s` for non-const, non-global variables. Also I am confused as to how many style guides we have now..

Reply:
oooh I looked at the solidity style guide :laughing: